iocage_lib/ioc_common.py

```diff
@@ -1078,3 +1078,42 @@ def get_host_gateways():
             gateways[af_mapping[af]]['interface'] = \
                 default_route[0]['interface-name']
     return gateways
+
+
+def parse_nat_fwds(forwards, log=None):
```

Comment:
Passing log makes no sense, this isn't a common operation. This is exclusive to nat, it belongs in that module.